src/V3EmitCModel.cpp

```diff
@@ -234,6 +234,8 @@ class EmitCModel final : public EmitCFunc {
         puts("const char* hierName() const override final;\n");
         puts("const char* modelName() const override final;\n");
         puts("unsigned threads() const override final;\n");
+        puts("void prepareClone() const;\n");
+        puts("void atClone() const;\n");
```

Comment:
Please puts a comment on how these are used.  Also somewhere in docs/guide/connecting.rst please describe this feature.